pallets/parachain-staking/src/lib.rs

```diff
@@ -2027,6 +2059,12 @@ pub mod pallet {
 					if remaining.is_zero() {
 						<DelegatorState<T>>::remove(&bond.owner);
 					} else {
+						if let Some(request) = delegator.requests.requests.remove(&candidate) {
+							delegator.requests.less_total -= request.amount;
```

Comment:
Same thing, I know its probably not necessary but just in case we have inconsistencies between storage items it would be good to use saturated math

Reply:
or checked math